Include/ceval.h

```diff
@@ -87,8 +87,8 @@ PyAPI_FUNC(void) Py_SetRecursionLimit(int);
 PyAPI_FUNC(int) Py_GetRecursionLimit(void);
 
 #if !PY_DEBUGGING_CHECKS
-#define Py_EnterRecursiveCall(where) (0)
-#define Py_LeaveRecursiveCall(where) ((void)0)
+#define Py_EnterRecursiveCall(...) (0)
+#define Py_LeaveRecursiveCall(..) ((void)0)
```

Comment:
Errors out on wrong number of arguments to the macro on MSVC.
The two dots is my mistake

Reply:
What I meant was "why is a difference from CPython needed here", and it looks like the answer is because I messed up the conversion of Py_LeaveRecursiveCall. I think the proper solution here is to keep the definition of Py_EnterRecursiveCall and to remove the erroneous argument from Py_LeaveRecursiveCall